### PR TITLE
Drop control message Scala wrapper - WorkerLost/WorkerLostResponse

### PR DESCRIPTION
**Pull request by pan3793**
### How was this patch tested?

/cc @related-reviewer

/assign @main-reviewer

### CODE REVIEW

File: server-master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala [hunk 1]
```diff
@@ -193,7 +193,13 @@ private[celeborn] class Master(
       executeWithLeaderChecker(null, timeoutDeadWorkers())
     case CheckForApplicationTimeOut =>
       executeWithLeaderChecker(null, timeoutDeadApplications())
-    case WorkerLost(host, rpcPort, pushPort, fetchPort, replicatePort, requestId) =>
+    case pb: PbWorkerLost =>
+      val host = pb.getRequestId
```
Comment (waitinfuture) [inline server-master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala:197]: pb.getHost

Comment (pan3793) [reply server-master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala]: fixed, but I can't believe UT passed w/ such error... we really should improve UT

Comment (waitinfuture) [reply server-master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala]: +1

**Review by waitinfuture — APPROVED** [review]
LGTM, thanks!

**Pull request merged**